addons/portal/static/src/chatter/portal/chatter_patch.js

```diff
@@ -3,12 +3,14 @@ import { Chatter } from "@mail/chatter/web_portal/chatter";
 import { patch } from "@web/core/utils/patch";
 import { useRef, onWillPatch, useEffect } from "@odoo/owl";
 
+Chatter.template = "portal.Chatter";
```

Comment:
Not for this PR but not clean to override the template, we should have a different component instead and better share common code/sub comments.